### ISSUE TITLE
Missing binding for KeyboardEventInit.Key

### ISSUE THREAD

**Issue by Porl91** [bridgedotnet/Bridge#2997]
The KeyboardEventInit class should have a property called 'Key':

Binding:
https://github.com/bridgedotnet/Bridge/blob/71188a59cf5a588dd1ddd4c78f3114e000dcb787/Html5/Events/KeyboardEvent.cs#L1236

MDN here:
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/KeyboardEvent

**Comment by geoffreymcgill** [comment]
Hi @Porl91 – Thanks for pointing out the missing Fields. There were a few more. I've added the the following to the **KeyboardEventInit** class. 

```csharp
/// <summary>
/// optional and defaulting to "", of type DOMString, that sets the value of KeyboardEvent.key.
/// </summary>
public string Key;

/// <summary>
/// optional and defaulting to "", of type DOMString, that sets the value of KeyboardEvent.code.
/// </summary>
public string Code;

/// <summary>
/// optional and defaulting to 0, of type unsigned long, that sets the value of KeyboardEvent.location.
/// </summary>
public long Location;

/// <summary>
/// optional and defaulting to false, of type Boolean, that sets the value of KeyboardEvent.repeat.
/// </summary>
public bool Repeat;

/// <summary>
/// optional and defaulting to false, of type Boolean, that sets the value of KeyboardEvent.isComposing.
/// </summary>
public bool IsComposing;

/// <summary>
/// optional and defaulting to 0, of type unsigned long, that sets the value of the deprecated KeyboardEvent.which.
/// </summary>
public long Which;
```

These new Fields will be included in the 16.1 release.

**Comment by Porl91** [comment]
@geoffreymcgill That's great. Thanks! :)
